metadata.md

```diff
@@ -1,35 +1,99 @@
 # Metadata for files
 
-There are two databases that are described below. Data is stored in /data. 
-- Wastewater sample results (wastewater_covid-19.csv)
-- Wastewater site information (wastewater_site.csv)
+There are three databases that are described below. Data is stored in [data](data). 
+
+- [Wastewater sample results](#test_results) ([wastewater_covid-19.csv](data/wastewater_virus.csv))
+- [Wastewater site information](#site) ([wastewater_site.csv](data/wastewater_site.csv)) 
+- [Wastewater laboratory information](#lab) (wastewater_lab.csv - **TBA**)
+
+For discussion: Is a database of assay information needed? Should assay methods be recorded and described, or does the site information suffice and remain unchanged?
+
+Entity Relationship Diagram [here](#erd).
 
 ### File naming convention
 - **date**: MM/DD/YYYY HH:mm:ss  (24 hour format, in UTC).
 - **location** TBD
 
-## Wastewater singnal results (wastewater_covid-19.csv)
+## SARS-CoV-2 test results (wastewater_covid-19.csv) <span id="test_results"><span>
+
+Results for a single SARS-CoV-2 wastewater test.
+
+- **sampleID**: Unique identification for sample. Suggest siteID-date-sample#. e.g. Ottawa1_2020-10-10-1 (Primary Key)
 - **sampleDate**: Date wastewater sample collected. End date for continuous testing. Additional location information in `wastewater_site.csv`.
-- **locationID**: Identifier for location where wastewater sample was taken. 
-- **locationName**: Name corresponding to `locationName`: location name cojld be a treatment plant, campus, institution or sewer location, etc.
-- **sampleID**: Unique sample identifier; locationID appended with sampleDate.
-- **N1_PMMV_mean**:	mean SARS-CoV-2 N1 gene region standardized to Pepper mild mottle virus (PMMV) (mean SARV-CoV-2 copies per copies of PMMV).
-- **N1_PMMV_sd**:	standard deviation of `N1_PMMV_mean`.
-- **N2_PPMV_mean**:	mean SARS-CoV-2 N2 gene region standardized to Pepper mild mottle virus (PMMV) (mean SARV-CoV-2 copies per copies of PMMV).
-- **N2_PPM_sd**: standard deviation of `N2_PMMV_mean`.
-- **PMMV_Ct**: TBD
-
-## Wastewater site information (wastewater_site.csv)
-- **locationID**:	Identifier for location where wastewater sample was taken. `locationID` is the same in `wastewater_covid-19.csv`.
-- **locationName**:	Name corresponding to `locationName`: location name cojld be a treatment plant, campus, institution or sewer location, etc. Additional location information in `wastewater_site.csv`.
-- **sampleType**: Type of wastewater sample: 
-  - sludge - 
-- **catchmentPop**:	Number of people represented in the wastewater sample.
-- **assayMethod**: Method used to assay viral signals.
-- **creator**:	Person 
-- **creatorEmail**:	
-- **maintainer**:	
-- **description**:
-- **notes**:
+- **siteID**: Identifier for location where wastewater sample was taken. Additional location information in `wastewater_site.csv`.
```

Comment:
Note change in metadata from `locationID` to `siteID`.

Reply:
Name change to use more commonly used term by Ottawa WW program. There is a wide variation in the term for 'site' or 'location' in WBE worldwide. Probably just need to make and decision and go with it.

Reply:
`siteID` works for me.